### DISCUSSION TITLE
Get data using experimental version nextjs13

### DISCUSSION THREAD

**Discussion by softwarepark** [swellstores/community#83] [Help] [answered]
Hi, I try get data (list of products) using experimental appDir in Nextjs. Before I using getStaticProps with swell sdk. This solution works. But now in experimental version Nextjs we don’t have getStaticProps

**Comment by maarten2424** [comment, marked as answer]
Hey @softwarepark 
If you are using the experimental `app` directory in Next.js, all of the components inside the app folder are server components by default. This means you can create your component including your data async like so:

```
export default async function Page() {
  const products = await swell.products.list({
    limit: 25,
    page: 1
  })
  return (
    <code>
      {products}
    </code>
  );
}
```

Hopefully this answers your question!
